tests/containerd/Dockerfile

```diff
@@ -5,18 +5,19 @@ ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/root/go/b
 
 RUN apt-get update \
     && apt-get -y upgrade \
-    && apt-get install -y bash golang libbtrfs-dev libnl-3-dev libnet1-dev \
+    && apt-get install -y bash golang-1.13 libbtrfs-dev libnl-3-dev libnet1-dev \
```

Comment:
Why are we using old/unsupported go version?

Reply:
that is the latest I've found available on ubuntu:eoan.  Do you suggest installing it manually?